pyproject.toml

```diff
@@ -0,0 +1,80 @@
+[tool.poetry]
+name = "django-serializable-model"
+version = "0.0.5"
+description = "Django classes to make your models, managers, and querysets serializable, with built-in support for related objects in ~100 LoC"
+readme = "README.md"
+
+packages = [
+    { include = "django_serializable_model.py" },
+    # this is the original, unintended name, and should be removed in the
+    # first breaking/major release, v1.0.0. See `serializable.py` comment.
+    { include = "serializable.py" }
+]
+include = [
+    "LICENSE",
+    "CHANGELOG.md"
+]
+
+authors = ["Anton Gilgur"]
+license = "Apache-2.0"
+homepage = "https://github.com/agilgur5/django-serializable-model"
+repository = "https://github.com/agilgur5/django-serializable-model"
+documentation = "https://github.com/agilgur5/django-serializable-model"
+
+classifiers=[
+    "Natural Language :: English",
+    "Intended Audience :: Developers",
+    "Topic :: Software Development :: Libraries",
+    "License :: OSI Approved :: Apache Software License",
+
+    "Development Status :: 3 - Alpha",
+    "Operating System :: OS Independent",
+    "Programming Language :: Python :: 2",
+    "Programming Language :: Python :: 2.7",
+    "Programming Language :: Python :: 3",
+    "Framework :: Django",
+    "Framework :: Django :: 1.4",
+    "Framework :: Django :: 1.5",
+    "Framework :: Django :: 1.6",
+    "Framework :: Django :: 1.7",
+    "Framework :: Django :: 1.8",
+    "Framework :: Django :: 1.9",
+    "Framework :: Django :: 1.10",
+    "Framework :: Django :: 1.11",
+    "Framework :: Django :: 2.0",
+    "Framework :: Django :: 2.1",
+    "Framework :: Django :: 2.2"
+]
+keywords=[
+    "django",
+    "serializer",
+    "serializers",
+    "serializer-django",
+    "serialize",
+    "json",
+    "dict",
+    "queryset",
+    "model",
+    "modelmanager",
+    "full",
+    "wadofstuff"
+]
+
+[tool.poetry.urls]
+"Tracker" = "https://github.com/agilgur5/django-serializable-model/issues"
+
+[tool.poetry.dependencies]
+python = "^3.5"
+
+[tool.poetry.dev-dependencies]
+django = "^2.2"
+pytest = [
+    {version = "^5.1", python = "^3.5"},
+    {version = "<5", python = "^2.7"}
+]
```

Comment:
well this was accidentally committed at some point welp... But how did this pass CI when it fails locally due to the Poetry bug???

Reply:
Seems like it works locally too, even though Python^2.7 isn't supported in deps... But if I add an `^2.7 ||` to the Python dep, I get the `SolverProblemError` as expected from before. Weird... seems like I might have found one more Poetry bug.